edx_proctoring/__init__.py

```diff
@@ -1,3 +1,9 @@
 """
-Init file for the top level edx_proctoring directory
+The exam proctoring subsystem for the Open edX platform.
 """
+
+from __future__ import absolute_import
+
+__version__ = '0.1.0'
```

Comment:
Great catch. I've updated this to be the true version number, and fixed setup.py to use it.